### DISCUSSION TITLE
How do I get started with using GitHub Copilot?

### DISCUSSION THREAD

**Discussion by anchalchaturvedi08** [community/community#143386] [Copilot Conversations] [answered]
### Select Topic Area

Question

### I am a CSE undergraduate student and I recently heard about GitHub Copilot. I'm interested in using it to enhance my coding productivity.

1. What are the steps to set up GitHub Copilot in my development environment?
2. Does it work with popular code editors like VS Code?
3. Are there any beginner-friendly tutorials or resources for learning how to use GitHub Copilot effectively?

**Comment by davevad93** [comment]
Hi @anchalchaturvedi08 ,

If you have an active `Copilot` subscription it's easy to set up in `VS Code`, simply install the `GitHub Copilot` extension, then login to GitHub with the same account that has the subscription.

For more information about your questions check out the official documentation:

1. [Set up GitHub Copilot in VS Code](https://code.visualstudio.com/docs/copilot/setup)
2. [Quickstart for GitHub Copilot](https://docs.github.com/en/copilot/quickstart?tool=vscode)
3. [Getting code suggestions in your IDE with GitHub Copilot](https://docs.github.com/en/copilot/using-github-copilot/getting-code-suggestions-in-your-ide-with-github-copilot)

**Comment by aminefjr** [comment, marked as answer]
Hi @anchalchaturvedi08 , You can also get it for free with your student status : https://education.github.com/pack

**Comment by anchalchaturvedi08** [comment]
Thank you both for your helpful responses! The information you provided about getting started with GitHub Copilot is incredibly valuable. I appreciate the detailed setup instructions and the tip about accessing it for free with my student status. Both answers have been very helpful!

**Reply by davevad93** [reply]
You're welcome @anchalchaturvedi08 , have a nice day! 👍
